pkg/services/authn/identity.go

```diff
@@ -21,10 +22,11 @@ var _ identity.Requester = (*Identity)(nil)
 
 type Identity struct {
 	// ID is the unique identifier for the entity in the Grafana database.
-	// If the entity is not found in the DB or this entity is non-persistent, this field will be empty.
-	ID identity.TypedID
+	ID string
```

Comment:
```suggestion
	ID int64
```
is there any non-numeric case where we use ID?